README.md

```diff
@@ -17,7 +17,7 @@ A demo of the interface can be found [on youtube][YT demo].
 Getting started with ansible-navigator is as simple as:
 
 ```
-pip install ansible-navigator
+pip install ansible-navigator[ansible-core]
```

Comment:
Use `pip3` and add double quotes as use of brackets does not work in all shells, like `pip3 install "ansible-navigator[ansible-core]"` -- safer to copy/paste ;)

Reply:
Fair enough. Although, I'd use single quotes then, to prevent interpolation if somebody accidentally copy-pastes vars inside...
```suggestion
pip3 install 'ansible-navigator[ansible-core]'
```